packages/ui-extensions-react/src/use-template-variable.ts

```diff
@@ -0,0 +1,99 @@
+import * as uiExtensionsSDK from '@datadog/ui-extensions-sdk';
+import * as React from 'react';
+
+import * as useContext from './use-context';
+
+function findTemplateVariableValue(
+    variableName: string,
+    templateVariables: uiExtensionsSDK.TemplateVariableValue[]
+): string | undefined {
+    const templateVariable = templateVariables.find(
+        (value: uiExtensionsSDK.TemplateVariableValue): boolean => {
+            return value.name === variableName;
+        }
+    );
+
+    if (templateVariable?.value == null) {
+        return;
+    }
+
+    return templateVariable.value;
+}
+
+/**
+ * Returns the current value of a specific template variable.
+ * This will be updated whenever the template variable change.
```

Comment:
Nit: should this be the following?

```suggestion
 * This will be updated whenever the template variable changes.
```